tests/test_strace.py

```diff
@@ -106,6 +110,13 @@ def test_socket(self):
             "import socket; socket.socket(socket.AF_INET,socket.SOCK_STREAM).close()",
             br'^socket\(AF_INET, SOCK_STREAM(\|SOCK_CLOEXEC)?')
 
+    @unittest.skipIf(UNTESTED, 'Untested system/OS')
+    def test_crash(self):
+        dn = os.path.join(os.path.dirname(__file__), 'crash')
+        shell = shutil.which('bash')
```

Comment:
Well, that's one reason to not rely on a shell, to be able to run tests even if bash is not available ;-)

Reply:
Ok, but this test relies on having a proper toolchain (at least gcc, make, ...) so bash is a weak dependency. Or maybe if the directory `tests/crash` is present but not used by any test (dead code) it should be removed ?